app/exporters/shopify/product_exporter.rb

```diff
@@ -0,0 +1,43 @@
+module Shopify
+  class ProductExporter
+    include Spree::Retail::PresenterHelper
+
+    def initialize(spree_product_id:, product_klass: Spree::Product,
+                   product_api: ShopifyAPI::Product,
+                   product_converter: Shopify::ProductConverter)
+
+      @spree_product = product_klass.find(spree_product_id)
+      @product_converter = product_converter
+      @product_api = product_api
+    end
+
+    def perform
+      shopify_product = find_shopify_product_for(spree_product)
```

Comment:
Should the following three lines be wrapped in 1 private method? Therefore the perform calls into the private methods and then has a very clear 'return' which we already have implemented.